mesheryctl/internal/cli/root/version.go

```diff
@@ -65,37 +53,57 @@ var versionCmd = &cobra.Command{
 		build := mctlCfg.GetVersion().GetBuild()
 		commitsha := mctlCfg.GetVersion().GetCommitSHA()
 
+		version := cfg.Version{
+			Build:     "unavailable",
+			CommitSHA: "unavailable",
+		}
+
 		logrus.Infof("Client Version: %v \t  GitSHA: %v", build, commitsha)
 
 		req, err := http.NewRequest("GET", fmt.Sprintf("%s/server/version", url), nil)
-		if requestErr(err, url) {
+		if err != nil {
+			logrus.Infof("Server Version: %v \t  GitSHA: %v", version.Build, version.CommitSHA)
```

Comment:
Will you rephrase to make this message more human-readable?

Reply:
More human readable? This is the error response which was earlier too 
```sh
Server Version: unavailable 	  GitSHA: unavailable

Unable to communicate with Meshery: Get "http://localhost:9081/api/server/version": dial tcp [::1]:9081: connect: connection refused
Ensure that Meshery is available, see Meshery Documentation (https://docs.meshery.io) for more help.
```

Reply:
Thank you, sir. Let's move forward with this.